notebooks/search/05-query-rules.ipynb

```diff
@@ -0,0 +1,437 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "87773ce7",
+      "metadata": {
+        "id": "87773ce7"
+      },
+      "source": [
+        "# Search using query rules\n",
+        "\n",
+        "<a target=\"_blank\" href=\"https://colab.research.google.com/github/kderusso/elasticsearch-labs/blob/kderusso/query-rules-notebook/notebooks/search/05-query-rules.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```

Comment:
Note: I will update these links before merging, but the link to my repo is required to run in colab.